src/Marten/Marten.csproj

```diff
@@ -31,6 +31,7 @@
         <PackageReference Include="System.Threading.Tasks.Dataflow" Version="4.11.1" />
         <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.7.0" />
         <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="[2.1.0, 6.0.0]" />
+        <PackageReference Include="System.Text.Json" Version="5.0.0" />
```

Comment:
We'll make this an add on lib, right? Not be in the box?

Reply:
This we need to discuss - in .NET 5 it's already built-in. I added NuGet, as built-in version in .NET Core 3.1 is too old and missing features. We can distribute that as separate package, but then maintainance might be harder as we'll have growing start of packages (NodaTime, Console, etc.).